FakeServer/GraphQL/GraphQLMiddleware.cs

```diff
@@ -44,25 +46,39 @@ public GraphQLMiddleware(RequestDelegate next, IDataStore datastore, IMessageBus
                 return;
             }
 
-            if (context.Request.Method != "POST" || !context.Request.ContentType.Contains("application/graphql"))
+            if (context.Request.Method != "POST" ||
+                !context.Request.ContentType.ContainsAny("application/graphql", "application/json"))
             {
                 context.Response.StatusCode = (int)HttpStatusCode.NotImplemented;
                 await context.Response.WriteAsync(JsonConvert.SerializeObject(new { errors = new[] { "Not implemented" } }));
                 return;
             }
 
-            var query = string.Empty;
-
-            using (var streamReader = new StreamReader(context.Request.Body))
+            var result = default(GraphQLResult);
```

Comment:
With reference types it is better to use null than default. Default is a good option with generics, where type is not known.